docs/site/Fields-filter.md

```diff
@@ -0,0 +1,83 @@
+---
+title: 'Fields filter'
+lang: en
+keywords: LoopBack 4.0, LoopBack 4, fields
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Fields-filter.html
+---
+
+A _fields_ filter specifies properties (fields) to include or exclude from the results.
+
+### Node.js API
+
+{% include content/angular-methods-caveat.html lang=page.lang %}
+
+<pre>
+{ fields: {<i>propertyName</i>: <true|false>, <i>propertyName</i>: <true|false>, ... } }
+</pre>
+
+Where:
+
+- _propertyName_ is the name of the property (field) to include or exclude.
+- `<true|false>` signifies either `true` or `false` Boolean literal. Use `true` to include the property or `false` to exclude it from results.
+
+By default, queries return all model properties in results. However, if you specify at least one fields filter with a value of `true`,
+then by default the query will include **only** those you specifically include with filters.
+
+### REST API
+
+{% include warning.html content="
+As a known bug, the option `<false>` does not work for url for now. Please include the properties you need to meet your requirement. The bug is tracked in the GH issue [#4992](https://github.com/strongloop/loopback-next/issues/4992" %}
+
+<pre>
+filter[fields][<i>propertyName</i>]=<true>&filter[fields][<i>propertyName</i>]=<true>...
+</pre>
+
+Note that to include more than one field in REST, use multiple filters.
+
+You can also use [stringified JSON format](Querying-data.html#using-stringified-json-in-rest-queries) in a REST query.
+
```

Comment:
Can we add a URL example of stringified JSON format for `fields` filter here? It would be very helpful for our readers to see an example right here without having to navigate away to the link where they will actually see an example for the `where` filter. Let the link be there, though. Small thing, but makes a big difference.

Reply:
Thanks for reminding me! I just realized that the url works well with the stringified JSON ( it has a bug with pure url, see https://github.com/strongloop/loopback-next/issues/4992). It will be a good workaround for `fields`!